src/tests/functional/plugin/cpu/single_layer_tests/fake_quantize.cpp

```diff
@@ -1,427 +1,385 @@
+
+        result << "LOW_BOUNDS=" << inDataLowBounds << "_";
+        result << "HIGH_BOUNDS=" << inDataHighBounds << "_";
+        result << "IL=" << CommonTestUtils::vec2str(inputLow) << "_";
+        result << "IH=" << CommonTestUtils::vec2str(inputHigh) << "_";
+        result << "OL=" << CommonTestUtils::vec2str(outputLow) << "_";
+        result << "OH=" << CommonTestUtils::vec2str(outputHigh) << "_";
+        result << "LEVELS=" << levels;
+
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+
+        return result.str();
+    }
+
+
+
+protected:
+    std::string layerName;
+
+    void SetUp() override {
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        fqSpecificParams fqParams;
+        inputShapes testShapes;
+        Precision inPrec;
+        std::pair<std::vector<float>, std::vector<float>> inputRangesValues;
+        bool shouldBeDecomposed;
+        CPUSpecificParams cpuParams;
+        std::tie(fqParams, testShapes, inPrec, inputRangesValues, shouldBeDecomposed, cpuParams) = this->GetParam();
+
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+
+        InputShape shapes;
+        std::vector<SizeVector> ranges;
+        std::tie(shapes, ranges) = testShapes;
+
+        inputDynamicShapes.push_back(shapes.first);
+        for (size_t i = 0; i < shapes.second.size(); i++) {
+            targetStaticShapes.push_back(std::vector<ov::Shape>{shapes.second[i]});
+        }
+
+        size_t levels;
+        std::vector<std::vector<float>> rangesBounds(RANGES_INPUT_NUMBER);
+        rangesBounds[0] = inputRangesValues.first;
+        rangesBounds[1] = inputRangesValues.second;
+        std::tie(inDataLowBounds, inDataHighBounds, rangesBounds[2], rangesBounds[3], levels) = fqParams;
+
+        auto ngInPrec = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inPrec);
+        ParameterVector params = builder::makeDynamicParams(ngInPrec, inputDynamicShapes);
+        auto paramOuts = helpers::convert2OutputVector(helpers::castOps2Nodes<opset5::Parameter>(params));
+
+        auto il = builder::makeConstant(ngInPrec, ranges[0], rangesBounds[0], rangesBounds[0].empty());
+        auto ih = builder::makeConstant(ngInPrec, ranges[1], rangesBounds[1], rangesBounds[1].empty());
+        auto ol = builder::makeConstant(ngInPrec, ranges[2], rangesBounds[2], rangesBounds[2].empty());
+        auto oh = builder::makeConstant(ngInPrec, ranges[3], rangesBounds[3], rangesBounds[3].empty());
+        auto fq = std::make_shared<opset5::FakeQuantize>(paramOuts[0], il, ih, ol, oh, levels);
+
+        layerName = shouldBeDecomposed ? "" : "FakeQuantize";
+
+        if (selectedType.empty()) {
+           selectedType = getPrimitiveType() + "_" + inPrec.name();
+        }
+
+        function = makeNgraphFunction(ngInPrec, params, fq, "FakeQuantizeCPU");
+    }
+
+private:
+    const size_t RANGES_INPUT_NUMBER = 4;
+
+    int64_t inDataLowBounds, inDataHighBounds;
```

Comment:
Previously I think this could be covered by the SubgraphBaseTest::run(), but I thinks it' better to override `generate_inputs` function since we have upper_bound & lower_bound for input data